datafusion/tests/sql/mod.rs

```diff
@@ -693,6 +695,21 @@ fn make_timestamp_nano_table() -> Result<Arc<MemTable>> {
     make_timestamp_table::<TimestampNanosecondType>()
 }
 
+/// Return a new table provider that has a single Int32 column with
+/// values between `seq_start` and `seq_end`
+pub fn table_with_sequence(
```

Comment:
this is now duplicated with the code in datafusion/src/test which is only compiled in test mode and thus can't be used from integration tests such as `sql_integration`

I think ideally we would move the contents of `datafusion/src/test` into its own `test_helpers` *crate* or something so it could be shared 🤔

Reply:
I like that and just use it in dev-dependencies where necessary.

Reply:
I actually just tried to do this and I hit the issue that the test library would have to depend on DataFusion but then datafusion would depend on the test library resulting in a circular dependency and cargo sadness.

So my plan will be to continue cutting stuff out of context.rs and then aim eventually to remove all of `datafusion/src/test` (either to its own crate or elsewhere)